src/main/java/uk/gov/ons/census/fwmtadapter/healthcheck/HeathCheck.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.ons.census.fwmtadapter.healthcheck;
```

Comment:
filename isn't spelled correctly, should be HealthCheck.java not HeathCheck.java

Reply:
Fixed. Fixed yesterday actually but forgot to push.